The interpreter now correctly issues :c:data:`PyRefTracer_DESTROY` for
deallocated objects.
